src/module/iotjs_module_fs.cpp

```diff
@@ -139,8 +138,16 @@ JHANDLER_FUNCTION(Open) {
   if (handler.GetArgLength() > 3 && handler.GetArg(3)->IsFunction()) {
     FS_ASYNC(env, open, handler.GetArg(3), path.data(), flags, mode);
   } else {
+    /* FIXME it is workaround of valgrind error on NULL value as 'stat' input */
+    /* Empty string should point some valid empty string */
+    char tmp[1] = { '0' };
```

Comment:
What is the valgrind issue here? And why not `{' \0 '}`? It would be great to give a bettername instead of `tmp`